flexmeasures/data/schemas/scheduling/storage.py

```diff
@@ -309,6 +350,68 @@ class DBStorageFlexModelSchema(Schema):
         validate=validate.Length(min=1),
     )
 
+    soc_usage = fields.List(
+        VariableQuantityField("MW"),
```

Comment:
I have a question here (and this applies to `soc_gain` as well (which is defined in the base branch for modelling) - state of charge is an energy measure in my opinion, and therefore energy units like kWh and MWh should be used.

Why are we using "MW" here - any reason you can recall?